exercises/anagram/anagram_test.rb

```diff
@@ -3,119 +3,86 @@
 require 'minitest/autorun'
 require_relative 'anagram'
 
-# Test data version:
-# 6a886e0
+# Common test data version: 196fc1a
 class AnagramTest < Minitest::Test
   def test_no_matches
     # skip
-    detector = Anagram.new('diaper')
-    anagrams = detector.match(["hello", "world", "zombies", "pants"])
-    assert_equal [], anagrams
+    assert_equal [], Anagram.new('diaper').match(["hello", "world", "zombies", "pants"]).sort
   end
 
   def test_detects_simple_anagram
     skip
-    detector = Anagram.new('ant')
-    anagrams = detector.match(["tan", "stand", "at"])
-    assert_equal ["tan"], anagrams
+    assert_equal ["tan"], Anagram.new('ant').match(["tan", "stand", "at"]).sort
   end
 
   def test_does_not_detect_false_positives
     skip
-    detector = Anagram.new('galea')
-    anagrams = detector.match(["eagle"])
-    assert_equal [], anagrams
+    assert_equal [], Anagram.new('galea').match(["eagle"]).sort
   end
 
-  def test_detects_multiple_anagrams
+  def test_detects_two_anagrams
     skip
-    detector = Anagram.new('master')
-    anagrams = detector.match(["stream", "pigeon", "maters"])
-    assert_equal ["maters", "stream"], anagrams.sort
+    assert_equal ["maters", "stream"], Anagram.new('master').match(["stream", "pigeon", "maters"]).sort
```

Comment:
ROFL! 

The usual 3 line pattern for tests is 

    expected = ...
    actual = ...
    assert_equal expected, actual

These 3 liners cause cognitive friction for me because they are 3 liners that violate that pattern.